code/modules/modular_computers/computers/item/disks/role_disks.dm

```diff
@@ -121,13 +121,16 @@
 	icon_state = "datadisk6"
 	starting_programs = list(
 		/datum/computer_file/program/supermatter_monitor,
+		/datum/computer_file/program/atmosscan,
+		/datum/computer_file/program/alarm_monitor,
 	)
 
 /obj/item/computer_disk/atmos
 	name = "atmospheric technician data disk"
 	desc = "Removable disk used to download atmos-related tablet apps."
 	icon_state = "datadisk6"
 	starting_programs = list(
+		/datum/computer_file/program/supermatter_monitor,
```

Comment:
This is now just the same as the engineering data disk. Remove one and keep the other